librz/core/cmd_descs/cmd_descs.h

```diff
@@ -1374,6 +1374,8 @@ RZ_IPI RzCmdStatus rz_egg_reset_handler(RzCore *core, int argc, const char **arg
 RZ_IPI RzCmdStatus rz_egg_show_config_handler(RzCore *core, int argc, const char **argv);
 // "help"
 RZ_IPI RzCmdStatus rz_help_handler(RzCore *core, int argc, const char **argv);
+// "? <expr>"
+RZ_IPI void rz_help_calc_expr(RzCore* core, const char* input);
```

Comment:
You don't want to place functions here. This is an auto-generated file as it should be written at the head of the file. Place this declaration in another header like rz_core.h